woocommerce/woocommerce
# In-App: make in-app header sticky

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR makes the in-app header sticky so that it's visible at all times. The goal is to improve discoverability. 

Closes https://github.com/Automattic/woocommerce.com/issues/21566.

### How to test the changes in this Pull Request:



1. Checkout this branch and get a new build
2. Load [the in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=extensions&path=%2Fextensions) and scroll down. Confirm the header is now sticky. 
3. Interact with the header: click all the tabs and scroll down. Click on the user avatar and open menu. Connect and disconnect to WCCOM. 
4. Check if you see a double scrollbar, like show in the screenshot from PR [Suppress second scrollbar on WooCommerce > Extensions pages #40046](https://github.com/woocommerce/woocommerce/pull/40046). 
    - I had to delete that fix, to make `position: sticky` work. After deleting it, I didn't get a second scrollbar. Could be a platform difference (I'm on Linux + Firefox). So curious to see what you'll see. 
6. Please test around this issue.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Make In-App Marketplace header sticky to improve discoverability. 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### Screenshots
![image](https://github.com/user-attachments/assets/271999d2-f4db-40b5-96c1-34a577a98b61)
